doc/howto/cluster_manage.md

```diff
@@ -23,10 +23,13 @@ and
 lxc config set cluster.max_standby <n>
 ```
 
+(cluster-manage-delete-nodes)=
 ## Deleting nodes
 
 To cleanly delete a node from the cluster use `lxc cluster remove <node name>`.
```

Comment:
To cleanly delete a member from the cluster use `lxc cluster remove <member name>`.